modules/administration_manual/pages/configuration/files/encryption_configuration.adoc

```diff
@@ -265,7 +265,7 @@ Then, enable the user-key, using the following command:
 
 [source,console]
 ....
-occ encryption:select-encryption-type user-key
+occ encryption:select-encryption-type user-keys
```

Comment:
Please write the complete command:
``sudo -u www-data php occ encryption:select-encryption-type user-keys``
as we have that in all commands described in occ commands // occ apps commands

Reply:
Well said, @mmattel. That's something to add to the upcoming style guide. Thanks for contributing the PR @paurakhsharma. Very much appreciated.